src/Driver/Exception/IdentityColumnsNotSupported.php

```diff
@@ -13,8 +14,8 @@
  */
 final class IdentityColumnsNotSupported extends AbstractException
 {
-    public static function new(): self
+    public static function new(?Throwable $previous = null): self
```

Comment:
What's the use case for the `$previous` exception? It doesn't seem to be used anywhere.

Reply:
Correct. Cleaned up in 5ff918d2f00ab65c235836791f7ba7acafeb6312.